Bottom: 070ad4074ca24a78565b334b04451ba020429abe
Top:    9fe02cfc9a3804b9559a551581b75c028f9b91bd
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-29 04:12:49 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
